mautic/mautic
# [FEATURE] New Gmail lookalike email theme added

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" 
| Bug fix?                               | yno
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no





Description:
This PR introduces a new Email theme, that provides classic Gmail formatting.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Go to Channel Emails
3. Create new Email
4. Check if Gmail Theme is available
5. Test use
